Assets/Script/StaticVars.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class StaticVars : MonoBehaviour
+{
+
+    public static int score;
+    public static bool win;
+
+    public static void addPoints(int points)
+    {
+        if (points < 0 && score < -points)
+        {
+            score = 0;
+        }
+        else{
+            score += points;
+        }
```

Comment:
Creo que refactorearía este bloque a algo del estilo
```suggestion
        score = Math.Max(0, score + points);
```

Si no me equivoco, hay que agregar un `using System;` como import arriba

Reply:
Agregado 👍